Gems/Atom/RPI/Code/Source/RPI.Public/MeshDrawPacket.cpp

```diff
@@ -126,9 +150,36 @@ namespace AZ
 
         void MeshDrawPacket::ClearShaderOptions()
         {
+            m_needUpdate = m_shaderOptions.size() > 0;
             m_shaderOptions.clear();
         }
 
+        void MeshDrawPacket::SetEnableDraw(RHI::DrawListTag drawListTag, bool enableDraw)
+        {
+            if (drawListTag.IsNull())
+            {
+                return;
+            }
+            // set the bit to true (1) if it's enabled for drawing
```

Comment:
nit: The comment states that the following code is going to set it to true if it's enabled, but that kind of implies that it will only set it to true, not false. But the code is also going to set it to false if it's disabled.

Maybe something like: // set the bit to true (1) if it's enabled for drawing, false otherwise

Or just remove the comment. I think the code here is self explanatory, and the comment in the header clearly describes what the function does. You could also copy that header comment here, since it's more clearly written.